### DISCUSSION TITLE
GitHub Download Speed Is Very Slow In India

### DISCUSSION THREAD

**Discussion by yourcodebuddy-in** [community/community#143926] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My internet speed is around 100-200Mbps. And, I am using Jio 5G Connection. The downloads from other sites are using full internet speed. And the difference in Github and Other site download speed is too big.

![image](https://github.com/user-attachments/assets/612b25d6-e39f-4a89-9aaa-46e8e081ce77)
![image](https://github.com/user-attachments/assets/a251d017-2b4f-4f49-9e21-640bc887d6c4)
![image](https://github.com/user-attachments/assets/5f1179cc-c53a-414e-aaf2-919be072575d)

**Comment by yourcodebuddy-in** [comment]
Why should I use VPN if I have a strong connection with a speed of over 100MBPS? I want to fix the issue without using VPN.

**Comment by vishnu-17o7** [comment]
experiencing the same problem.  edit: use cloudflare 1.1.1.1 Seems like some sort of DNS issue.

**Reply by Poko-Cmnt** [reply]
Same here, any sort of vpns usually do the job. But why? Why should i must use vpn ?

**Comment by kerim** [comment]
I've noticed this as well. VPN performance is slightly better, but still much slower than other sites.
